docs/sources/flow/concepts/clustering.md

```diff
@@ -0,0 +1,303 @@
+---
```

Comment:
I'm a little worried the concept page is doing too much. 

From the Writers' Toolkit on [Concept topics](https://grafana.com/docs/writers-toolkit/writing-guide/topic-types/concept/#concept-topic):

> The following types of content can be included in concepts:
>
> * Detailed overviews of Grafana’s features with benefits and clearly defined terms
> * Diagrams that help users understand the components of a system
> * Process flow diagrams
> * Best practice guidelines
> * An example of how a feature is used. Examples might include screenshots or other supporting visuals
> 
> A concept topic does not include:
>
> * Step-by-step instructions
> * Reference information, such as lookup tables or lists of values

I would suggest splitting this doc up, where this focuses more on the benefits of clustering, and maybe some architectural diagrams, with best practices. 

I would move everything else to either the appropriate reference topic (i.e., documenting command line flags in cli/run.md) or troubleshooting information to the [Debugging](https://grafana.com/docs/agent/latest/flow/monitoring/debugging/) page.

@clayton-cornell Am I off-base here?

Reply:
Ok, I've went through a second iteration of this

a) The command-line flags with their explanation have been moved back to the `run` command docs
b) The debugging information is more high-level now and has been decoupled from mentions of the mixin dashboards; it mentions the new UI page and logs instead which will always be present
c) There's a new page just for the `mixin` with instructions for people who want to install it and showcasing the clustering dashboards
d) The concepts clustering page is now more streamlined with less 'empty' braggadocious statements. It highlights the target auto-distribution usecase first, and shows the Helm section more clearly as an example.

I'll look into whether there's any diagrams to be added or a best practices section, but let me know if there's any other high-level structure suggestions cc @mattdurham

Reply:
I would still like to see an example that doesn't depend on helm/k8s, so people know its not just for k8s. Even if its a one liner running the raw executable multiple times.

Reply:
@mattdurham I agree, but can we make that a follow up task for when Paschalis is back from PTO?